Makefile

```diff
@@ -73,6 +73,11 @@ lint: kondo cljfmt eastwood
 # Deployment is performed via CI by creating a git tag prefixed with "v".
 # Please do not deploy locally as it skips various measures.
 deploy: check-env clean
+	@if ! echo "$(CIRCLE_TAG)" | grep -q "^v"; then \
+		echo "CIRCLE_TAG: $(CIRCLE_TAG) must start with 'v'."; \
```

Comment:
Probably we can prefix this with `[Error]`, so it's easier to spot it in the CI logs.